### DISCUSSION TITLE
always() and f_always()

### DISCUSSION THREAD

**Discussion by jengelbertt** [ManimCommunity/manim#3861] [Q&A]
Please write an example of the use of always() and f_always(). I am completely confused. 
Are these two methods available via Mobject?

**Comment by JasonGrace2282** [comment]
`always` is supposed to be used like
```py
d = Dot()
sq = Square()
always(d.next_to, sq, UP)
```
`f_always` is supposed to be used like
```py
def calc_pos():
    sq.get_center()
sq = Square()
d = Dot()
f_always(d.move_to, calc_pos)
```

Admittedly, this is not very intuitive, so #3852 was introduced to make the syntax for `always` more like
```py
d = Dot()
d.always.next_to(sq, UP)
```
Hopefully, if the devs can find the time, it will be included in Manim v0.19.0
